packit_service/worker/jobs.py

```diff
@@ -55,24 +55,75 @@
 logger = logging.getLogger(__name__)
 
 
-def get_handlers_for_event(
-    event: Event, package_config: PackageConfig, packit_comment_command_prefix: str
+def get_handlers_triggered_by_job(
```

Comment:
I would be careful with using `job` here since we use it more for the implementation, e.g. `job_config`, `job_type`.

What about preserving the word `event`?

Also, it looks like this is only about comments and reruns.


What about something really explicit like `get_handlers_for_reaction_event` or `get_handlers_for_comment_and_rerun_event`?